strongbox-web-core/pom.xml

```diff
@@ -186,6 +204,12 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>strongbox-storage-maven-layout-provider</artifactId>
             <version>${project.version}</version>
+            <exclusions>
+              <exclusion>
+                <groupId>org.codehaus.groovy</groupId>
+                <artifactId>groovy-all</artifactId>
+              </exclusion>
+            </exclusions>
```

Comment:
Cosmetic: Could you please re-indent this with 4 spaces, as per our convention? Thanks!

Reply:
Fixed.

Reply:
Thanks! :)